thentos-core/src/Thentos/Sybil/Captcha.hs

```diff
@@ -0,0 +1,15 @@
+{-# LANGUAGE OverloadedStrings    #-}
+
+module Thentos.Sybil.Captcha where
```

Comment:
`Thentos.Sybil.Blacklists`, `Thentos.Sybil.Analytics`.  I think it should stay here.